tests/test_portfolio_optimization.py

```diff
@@ -139,7 +143,8 @@ def test_portfolio_AR():
     po_energy = qaoa_obj(x0).real
     po_ar = (po_energy - best_portfolio[1]) / (best_portfolio[0] - best_portfolio[1])
     # a problem with known AR > 0.7564
-    assert po_ar > 0.75
+    # a problem with known AR Yahoo >= 0.7349
```

Comment:
I think the comments are duplicated? Don't need both line 145 and 146

Reply:
Done!